api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,18 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`
+// The resource_names_subscribe field in DeltaDiscoveryRequest contains a list of virtual host names or aliases
+// to track. The contents of an alias would be the contents of a "host" or "authority" header used to make an
```

Comment:
Nit: the preferred RST formatting for literals such as header name is something like `*host*`

Reply:
fixed